src/main/resources/application.yaml

```
@@ -1,7 +1,20 @@
+files:
+  location:
+    blacklist: src/main/resources/customer_blacklist.csv
+
 ---
 spring.profiles.active: local
 
+spring:
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/spring_basic?serverTimezone=Asia/Seoul
+    username: root
+    password: 
```

```suggestion
    password: {DB_PASSWORD}
```
이런식으로 관리하는 방법도 알아보시면 좋을 것 같아요!

---

찾아보겠습니다! 그래도 모르겠으면 검색 키워드 좀 요청해도 될까요?

---

넵 ~ :)